### PR TITLE
app to api netdata cloud

### PR DESCRIPTION
**Pull request by underhood**
##### Summary
Changes all api accesses to `api.netdata.cloud` where application itself is mentioned keep `app.netdata.cloud`

Fixes https://github.com/netdata/netdata/issues/13682

##### Test Plan
The agent should still be able to connect to the production cloud. Documentation should be checked so that in places where the application is mentioned (e.g. sign in to cloud, link to cloud etc.) stays `app.netdata.cloud`

##### Additional Information

<details> <summary>For users: How does this change affect me?</summary>
  
</details>

### CODE REVIEW

**Comment by ilyam8** [comment]
<details>
<summary>app.netdata.cloud matches</summary>

```cmd
pve-deb-work nd_underhood (cloudurl_change *) $ git grep "app.netdata.cloud" -- ':(exclude)*.js' ':(exclude)*.map'
README.md:[Get Netdata Cloud now!](https://app.netdata.cloud/)
aclk/README.md:The Cloud App lives at app.netdata.cloud which currently resolves to the following list of IPs:
docs/contributing/style-guide.md:must reflect the _current state of [production](https://app.netdata.cloud).
docs/contributing/style-guide.md:| **Netdata Cloud**           | The web application hosted at [https://app.netdata.cloud](https://app.netdata.cloud) that helps you monitor an entire infrastructure of distributed systems in real time. <br /><br />Never use _Cloud_ without the preceding _Netdata_ to avoid ambiguity.                                                                                                                                                           |
docs/guides/deploy/ansible.md:-   A Netdata Cloud account. [Sign in and create one](https://app.netdata.cloud) if you don't have one already.
docs/guides/export/export-netdata-metrics-graphite.md:in](https://app.netdata.cloud) and get started for free. If you need some help along the way, read the [get started with
docs/guides/monitor/anomaly-detection-python.md:All these metrics and alarms are available for centralized monitoring in [Netdata Cloud](https://app.netdata.cloud). If
docs/guides/monitor/anomaly-detection-python.md:- A Netdata Cloud account. [Sign up](https://app.netdata.cloud) if you don't have one already.
docs/guides/monitor/kubernetes-k8s-netdata.md:-   A free Netdata Cloud account. [Sign up](https://app.netdata.cloud/sign-up?cloudRoute=/spaces) if you don't have one
docs/guides/monitor/kubernetes-k8s-netdata.md:To get started, [sign in](https://app.netdata.cloud/sign-in?cloudRoute=/spaces) to your Netdata Cloud account. Head over
docs/guides/monitor/lamp-stack.md:- Optionally, a [Netdata Cloud](https://app.netdata.cloud/sign-up?cloudRoute=/spaces) account, which you can use to view
docs/guides/monitor/lamp-stack.md:> If you [signed up](https://app.netdata.cloud/sign-up?cloudRoute=/spaces) for Netdata Cloud earlier, you can also view
docs/guides/monitor/lamp-stack.md:[sign up](https://app.netdata.cloud/sign-up?cloudRoute=/spaces) for a free Netdata Cloud account.
docs/guides/monitor/process.md:-   A Netdata Cloud account. [Sign up](https://app.netdata.cloud) if you don't have one already.
docs/guides/monitor/process.md:Cloud](https://app.netdata.cloud) for infrastructure monitoring, or single-node monitoring with the local Agent's
docs/guides/monitor/visualize-monitor-anomalies.md:In either [Netdata Cloud](https://app.netdata.cloud) or the local Agent dashboard at `http://NODE:19999`, click on the
docs/guides/monitor/visualize-monitor-anomalies.md:[Netdata Cloud](https://app.netdata.cloud) features a drag-and-drop [dashboard
docs/guides/step-by-step/step-03.md:<p><a href="https://app.netdata.cloud" className="button button--lg">Sign in to Cloud</a></p>
docs/guides/troubleshoot/monitor-debug-applications-ebpf.md:If you don't already have a Netdata Cloud account, go [sign in](https://app.netdata.cloud) and get started for free.
docs/quickstart/infrastructure.md:[Netdata Cloud](https://app.netdata.cloud) provides scalable infrastructure monitoring for any number of distributed
docs/quickstart/single-node.md:To see a node's dashboard in Netdata Cloud, [sign in](https://app.netdata.cloud). From the **Nodes** view in your
health/notifications/alarm-notify.sh.in:                        <img alt="Netdata Logo" height="auto" src="https://app.netdata.cloud/static/email/img/full_logo.png" style="border:0;display:block;outline:none;text-decoration:none;height:auto;width:100%;font-size:13px;" width="130">
health/notifications/alarm-notify.sh.in:                              <img height="auto" src="https://app.netdata.cloud/static/email/img/community_icon.png" style="border:0;display:block;outline:none;text-decoration:none;height:auto;width:100%;font-size:13px;" width="48">
health/notifications/alarm-notify.sh.in:                              <img height="auto" src="https://app.netdata.cloud/static/email/img/configure_icon.png" style="border:0;display:block;outline:none;text-decoration:none;height:auto;width:100%;font-size:13px;" width="48">
packaging/installer/methods/kickstart.md:1. Sign in to [Netdata Cloud](https://app.netdata.cloud/sign-in?cloudRoute=/spaces)
packaging/installer/methods/kubernetes.md:[connect](/claim/README.md) your Kubernetes cluster to [Netdata Cloud](https://app.netdata.cloud). The connection process securely
packaging/installer/methods/macos.md:Cloud](https://app.netdata.cloud/sign-in?cloudRoute=/spaces), then clicking on **Connect Nodes** in the [Spaces management
```

</details>

**Comment by underhood** [comment]
> 

We cannot replace it everywhere. In places where it refers to actual app it should be `app.netdata.cloud`

**Comment by ilyam8** [comment]
> We cannot replace it everywhere.

I didn't mean that at all. Just made [first comment](https://knowyourmeme.com/memes/first)!

**Comment by thiagoftsm** [comment]
@underhood let  me be the second (am I missing something @ilyam8 ?), when I am accessing `api.netdata.cloud` I am receiving the message `default backend - 404`. Should we set this PR as draft until the issue is fixed?

**Comment by underhood** [comment]
@thiagoftsm the web application did not move. It stays on `app.netdata.cloud`. `api.netdata.cloud` is for API calls only and it will not serve web application if you put that URL in browser. This is expected

**Review by thiagoftsm — APPROVED** [review]
After last answer I retested the PR and everything is working as expected. LGTM!

**Review by stelfrag — APPROVED** [review]

File: claim/README.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ When coming from  [Nodes view page](https://learn.netdata.cloud/docs/cloud/visua
 If you want to connect a node that is running on a Linux environment, the script that will be provided to you by Netdata Cloud is the [kickstart](/packaging/installer/README.md#automatic-one-line-installation-script) which will install the Netdata Agent on your node, if it isn't already installed, and connect the node to Netdata Cloud. It should be similar to:
 
 ```
-wget -O /tmp/netdata-kickstart.sh https://my-netdata.io/kickstart.sh && sh /tmp/netdata-kickstart.sh --claim-token TOKEN --claim-rooms ROOM1,ROOM2 --claim-url https://app.netdata.cloud
+wget -O /tmp/netdata-kickstart.sh https://my-netdata.io/kickstart.sh && sh /tmp/netdata-kickstart.sh --claim-token TOKEN --claim-rooms ROOM1,ROOM2 --claim-url https://api.netdata.cloud
```
Comment (ralphm) [inline claim/README.md:76]: Why do these examples have `--claim-url`? Wouldn't leaving this off work with a default?

File: packaging/installer/kickstart.sh [hunk 2]
```diff
@@ -27,7 +27,7 @@ KICKSTART_SOURCE="$(
 )"
 PACKAGES_SCRIPT="https://raw.githubusercontent.com/netdata/netdata/master/packaging/installer/install-required-packages.sh"
 PATH="${PATH}:/usr/local/bin:/usr/local/sbin"
-PUBLIC_CLOUD_URL="https://app.netdata.cloud"
+PUBLIC_CLOUD_URL="https://api.netdata.cloud"
```
Comment (ralphm) [inline packaging/installer/kickstart.sh:30]: I think this is not correct, as `PUBLIC_CLOUD_URL` is also used in this script to point to the web app in a `printf`.

**Pull request merged**